src/values/AbstractObjectValue.js

```diff
@@ -31,13 +31,6 @@ export default class AbstractObjectValue extends AbstractValue {
     super(realm, types, values, args, buildNode, optionalArgs);
   }
 
-  clone(): AbstractObjectValue {
-    let result = new AbstractObjectValue(this.$Realm, this.types, this.values, this.args, this._buildNode);
-    if (this.kind) result.kind = this.kind;
-    if (this.intrinsicName) result.intrinsicName = this.intrinsicName;
```

Comment:
kind and intrinsicName fields seems are not set in AbstractValue.createFromConditionalOp() method. Is this expected?

Reply:
The result of a conditional operation is never intrinsic, so that is expected. The kind is in fact set to "conditional".